.github/workflows/verify-dependabot.yml

```diff
@@ -0,0 +1,36 @@
+name: Verify (dependabot)
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
+jobs:
+  build:
+    if: github.actor == 'dependabot[bot]'
+    name: Verify Build
+    timeout-minutes: 5
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Did we intentionally drop all of the names and spacing between these steps?

Reply:
Yep, this was intentionally done to be concise.